Client/Frontend/Home/Pocket/FxPocketHomeHorizontalCell.swift

```diff
@@ -37,7 +37,7 @@ class FxPocketHomeHorizontalCell: UICollectionViewCell, ReusableCell {
         static let interGroupSpacing: CGFloat = 8
         static let generalCornerRadius: CGFloat = 12
         static let titleFontSize: CGFloat = 49 // Style subheadline - AX5
-        static let sponsoredFontSize: CGFloat = 49 // Style subheadline - AX5
+        static let sponsoredFontSize: CGFloat = 12 // Style subheadline - AX5
```

Comment:
Text size shouldn't be limited to 12 in this case, it should grow up to AX5

Reply:
We use DynamicFontHelper to set a text style and a max font size for accessibility feature (dynamic type). You should choose the text style according to the font size you need (also the text style is normally defined in the design). So from what I gather in your case the font style should be .caption1 since you expect a size of 12:

```
        label.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .caption1,
                                                                   maxSize: UX.sponsoredFontSize)
```

So by default your text size will be 12, as defined by Apple guidelines [here](https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/)

Then the max font size should be 43, since we want the text to be able to grow according to user preference (if they use dynamic type). AX5 is the name of the biggest Larger Accessibility Type Sizes.

```
        static let sponsoredFontSize: CGFloat = 43 // Style caption1 - AX5
```

<img width="877" alt="Screen Shot 2022-06-07 at 12 02 29 PM" src="https://user-images.githubusercontent.com/11338480/172428234-d474a722-37d9-4474-83d7-ea2979310936.png">

Let me know if that make sense!

Reply:
Thanks, updated 😁